### PR TITLE
[PROD-1295] Fix nimminers install

### PR DESCRIPTION
**Pull request by 0xyoung-nimble**
Changes:
- fix nimminers install problems
- fix miner run problems

### CODE REVIEW

**Comment by x-nimble** [comment]
$ python3 -m pip install -e .

```
ERROR: Ignored the following versions that require a different python version: 0.0.1 Requires-Python >=3.10; 0.0.3 Requires-Python >=3.10; 0.0.5 Requires-Python >=3.10; 0.0.6 Requires-Python >=3.10; 0.0.7 Requires-Python >=3.10; 0.0.8 Requires-Python >=3.10; 0.0.9 Requires-Python >=3.10; 8.19.0 Requires-Python >=3.10; 8.20.0 Requires-Python >=3.10; 8.21.0 Requires-Python >=3.10
ERROR: Could not find a version that satisfies the requirement nimcli==0.0.9 (from nimminers) (from versions: none)
ERROR: No matching distribution found for nimcli==0.0.9
```

File: scripts/run_miner.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ if [ -z "$CHAIN_ENDPOINT" ]; then
     exit 1
 fi
 
-python neurons/miners/nimbleLM/miner.py \
```
Comment (x-nimble) [inline scripts/run_miner.sh:28]: add `scripts/run_miner.sh ` usage to readme

File: setup.py [hunk 2]
```diff
@@ -66,8 +66,8 @@ def read_requirements(path):
 
 setup(
     # package names and code path config
-    name="nimmibers",
-    packages=['nimminers'],
+    name="nimminers",
+    # packages=['nimminers'],
```
Comment (x-nimble) [inline setup.py:70]: keep line 70 but remove line 78?

File: scripts/run_miner.sh (hunk 1 above)
Comment (0xyoung-nimble) [reply scripts/run_miner.sh:28]: We are using python3.10 or higher, and I updated README.md in [PROD-1307]

**Pull request merged**